src/share/util/shr_string_mod.F90

```diff
@@ -1237,7 +1237,8 @@ subroutine shr_string_listMerge(list1,list2,listout,rc)
     !EOP
 
     !----- local -----
-    character(SHR_KIND_CX) :: l1,l2   ! local char strings
+    character(len=len(list1)) :: l1
```

Comment:
Are you sure that this initialization is supported by all our current compilers?